src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs

```diff
@@ -244,6 +245,24 @@ static bool IsIncorrectDotNetObjectRefUse(JsonElement item, Type parameterType)
             }
         }
 
+        /// <summary>
+        /// Receives notification that a call from .NET to JS has finished, marking the
+        /// associated <see cref="Task"/> as completed.
+        /// </summary>
+        /// <param name="arguments">The serialized arguments for the callback completion.</param>
+        public static void EndInvoke(string arguments)
+        {
+            // We don't need to notify the client if a JS interop call fails to be processed on the server.
+            // The client needs to catch potential exceptions. We do it this way so that we can log errors
+            // processing the callback completion.
+            var parsedArgs = ParseArguments(
+                nameof(EndInvoke),
+                arguments,
+                EndInvokeParameterTypes);
+
+            EndInvoke((long)parsedArgs[0], (bool)parsedArgs[1], (JSAsyncCallResult)parsedArgs[2]);
```

Comment:
Can we remove the other overload of `EndInvoke`, or at least make it private and not have `[JSInvokable]`, so there's only one entrypoint? I think now that `endInvokeJSFromDotNet` is becoming part of the platform contract, it's the platform's job to have some way of calling `EndInvoke`, and this library doesn't need to provide any `[JSInvokable]` version of it.

I know that means that a naive implementation in WebAssembly will be double-serializing the args, but we're always free in the future to special case that so that WebAssembly calls `EndInvoke(string arguments)` directly without going through JS interop.

Reply:
I got rid of the [JSInvokable] on EndInvoke by providing a custom platform method inside Mono.WebAssembly.JSInterop

Reply:
done